### DISCUSSION TITLE
Additional information to store concerning sequences

### DISCUSSION THREAD

**Discussion by gwhitney** [numberscope/backscope#138] [Ideas]
Now that our database is becoming more robust, and has factorizations, for example, is there any additional information we should be generating/collecting/storing about each sequence? I am motivated to open this discussion because I think the following would be a really worthwhile feature to add:
Some OEIS sequences are relatively easy to calculate additional terms for. (Others would be a major research project to extend by even a single term.) For those which can be readily calculated, I think it would be good to collect and store the means for doing such calculations.
In particular, for whatever notation we ultimately support for sequences by "formula," it would be good to store the appropriate formula for those sequences that have them. Then the backend could support an arbitrary number of terms for those sequences. A nice example is the partial sums of the Kolakoski sequence translated to have conjectural zero mean (A088568). The [plot on oeis.org](https://oeis.org/A088568/graph) appears possibly to be "drifting down," though. However, that plot ends around index 10000, because typically the OEIS only stores about that many terms for a given sequence, with a few exceptions. But this sequence is readily calculable, so it would be nice to simply extend the plot to a  few more orders of magnitude of terms to see if that drift persists.

Now, I doubt whatever formula syntax we settle on will actually allow an expression for this sequence. But there is a Python program posted on the OEIS for A000002, and this sequence is easily expressed in terms of A000002. So we could imagine that for those sequences that don't have direct formulas in our formula language, but do have python programs, we could collect and store the python programs. Since the back end provides all of the sequence terms and it is running python anyway, it could then use the python program to supply as many terms as a visitor wants (CPU time permitting).

Are there other data/statistics about sequences that it's worth generating/keeping?

**Comment by katestange** [comment]
Some also have pari code. We also discussed this years ago near the beginning of the project, and I think it's a great long-term goal!

**Reply by gwhitney** [reply]
Ah, true, and we are running Pari, so that could potentially be helpful as well.
